A33-Fault-Injection.md

```diff
@@ -0,0 +1,253 @@
+Client-Side Fault Injection
+----
+* Author(s): lidiz
+* Approver: roth, ejona, dfawley
+* Status: In Review
+* Implemented in: all languages
+* Last updated: Feb 4th, 2021
+* Discussion at: https://groups.google.com/g/grpc-io/c/d9dI5Hy9zzk
+
+## Abstract
+
+Fault injection is an essential feature that assists developers in building fault-tolerant applications. To keep gRPC’s competitive advantage, this doc aims to design a fault injection mechanism on the client side for gRPC while allowing transparent transition from Envoy configuration.
+
+
+## Background
```

Comment:
I think most of the material in this section is redundant with what comes later in the doc.  I suggest removing most of it.

I think that all we really need in the "background" section is the following:

- The text you currently have in the "Envoy HTTPFilter" subsection, immediately below.
- A link to https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/fault_filter, as a reference for the Envoy fault injection filter functionality.